gtsfm/scene_optimizer.py

```diff
@@ -87,6 +87,43 @@ def __init__(self, matcher: MatcherBase, verifier: VerifierBase):
         self.matcher = matcher
         self.verifier = verifier
 
+    def __compute_metrics(
```

Comment:
Can we move this entirely to a different util file? 
This file will probably be the most frequently read in this repository, so I hope you agree that we should keep it very high level. 
I really like how this file is organized with very high-level classes, but it is already getting too big.

Reply:
I agree it is getting a bit too big -- maybe in a follow-up PR we should put each of TwoViewEstimator, MultiViewOptimizer into their own files

Reply:
If they go in their separate files, do we still move the functions to other files as Akshay recommends?

Reply:
@johnwlambert

Reply:
@ayushbaid up to you. I thought it might be cleaner, but it might be okay if they are in separate files, as long as we only have two function calls for metrics.

Reply:
We have moved to individual files and removed class functions to general functions in the same file.